docs/source/convert.rst

```diff
@@ -202,7 +202,7 @@ data attributes that are not recorded in the raw data files but need to be
 specified according to the SONAR-netCDF4 convention.
 These attributes are metadata and include
 ``platform_name``, ``platform_type``, ``platform_code_ICES``,
-and sometimes ``water_level``, depending on the sonar model.
+and sometimes ``range_offset``, depending on the sonar model.
```

Comment:
The comment I will make is not about the change to range_offset itself (which is correct), but rather a result of rereading the sentence with fresh eyes. @leewujung it seems like `range_offset` is actually a variable, not an attribute as described in this sentence. Should the reference to `range_offset` be removed from here altogether? In cases where `range_offset` is a constant, it would still be a variable, wouldn't it?

Reply:
I think you are correct, that `range_offset` is indeed a variable.